python/semantic_kernel/connectors/telemetry.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import os
+
+import pkg_resources
+
+TELEMETRY_DISABLED_ENV_VAR = "AZURE_TELEMETRY_DISABLED"
+
+IS_TELEMETRY_ENABLED = (
+    os.environ.get(TELEMETRY_DISABLED_ENV_VAR, "false").lower() != "true"
+)
+
+HTTP_USER_AGENT = "Semantic-Kernel"
+
+APP_INFO = {
+    "name": HTTP_USER_AGENT if IS_TELEMETRY_ENABLED else "",
+    "version": pkg_resources.get_distribution("semantic-kernel").version
+    if IS_TELEMETRY_ENABLED
+    else "",
+    "url": "",
+}
```

Comment:
I think this should be changed to:

```suggestion
APP_INFO = (
    {
        "name": HTTP_USER_AGENT,
        "version": pkg_resources.get_distribution("semantic-kernel").version,
        "url": "",
    }
    if IS_TELEMETRY_ENABLED
    else None
)
```

to avoid appending an empty space after the UserAgent when the telemetry is disabled